peps/pep-0769.rst

```diff
@@ -1,11 +1,12 @@
 PEP: 769
 Title: Add a 'default' keyword argument to 'attrgetter' and 'itemgetter'
 Author: Facundo Batista <facundo@taniquetil.com.ar>
+Discussions-To: https://discuss.python.org/t/76419
 Status: Draft
 Type: Standards Track
 Created: 22-Dec-2024
 Python-Version: 3.14
-
+Post-History: `07-Jan-2025 <https://discuss.python.org/t/76419>`__
```

Comment:
I'd suggest keeping the trailing comma in case you have future posts for a smaller `git diff`. It is stripped by the rendering system. This also keeps the newline
```suggestion
Post-History: `07-Jan-2025 <https://discuss.python.org/t/76419>`__,

```

Reply:
A trailing comma is a good thing in *multiline* lists (so adding a line does not change the previous one), but this field so far is "all in the same line", so there is no gain. Thanks anyway!